apollo-runtime/src/main/java/com/apollographql/apollo/internal/RealApolloWatcher.java

```diff
@@ -62,6 +62,10 @@
     cache.unsubscribe(recordChangeSubscriber);
   }
 
+  @Override public boolean isCanceled() {
+    return !isActive;
```

Comment:
nit: Mind changing the field `isActive` to `isCanceled`. Seems to better fit the model